### PR TITLE
Add more settings

### PR DESCRIPTION
**Pull request by gitlawr**

### CODE REVIEW

File: docs/settings/settings.md [hunk 1]
```diff
@@ -27,9 +41,22 @@ Default: none
 }
 ```
 
+## `cluster-registration-url`
+
+This setting allows you to import the Harvester cluster to Rancher for multi-cluster management.
```
Comment (nunix) [inline docs/settings/settings.md:46]:
```suggestion
This setting allows you to import the Harvester cluster to Rancher unified multi-cluster app management.
```

Comment (nunix) [reply docs/settings/settings.md:46]: rephrased based on the page https://rancher.com/why-rancher (Rancher unified multi-cluser app management)

**Review by nunix — COMMENTED** [review]
Rephrased the `cluster-registration-url` description

File: docs/settings/settings.md (hunk 1 above)
Comment (gitlawr) [reply docs/settings/settings.md:46]: This is for managing multiple Harvester clusters using Rancher(https://docs.harvesterhci.io/v0.3/rancher/virtualization-management/#virtualization-management). App management is not the focus in Harvester use case.

**Review by guangbochen — APPROVED** [review]
LGTM

**Pull request merged**